lighthouse-core/test/scripts/i18n/collect-strings-test.js

```diff
@@ -172,14 +172,14 @@ describe('Convert Message to Placeholder', () => {
   it('passthroughs a basic message unchanged', () => {
     const message = 'Hello World.';
     const res = collect.convertMessageToPlaceholders(message, undefined);
-    expect(res).toEqual({message, placeholders: undefined});
+    expect(res).toEqual({message, placeholders: {}});
```

Comment:
Arg.  This is less strict again :frowning: Now there isn't a great check for regressions unless someone looks at the `.ctc.json` files after generation. Not a showstopper, but I liked that this was caught in the test.

Reply:
> Arg. This is less strict again 😦 Now there isn't a great check for regressions unless someone looks at the `.ctc.json` files after generation. Not a showstopper, but I liked that this was caught in the test.

yeah, we can play with a way to make it caught, but this was part of my thinking on if we should check in `en-US.ctc.json` after all...